ios/brave-ios/fastlane/Fastfile

```diff
@@ -6,54 +6,25 @@ fastlane_version "2.86.0"
 
 default_platform :ios
 
-CONFIG_PATH = "../App/Configuration"
-LOCAL_PATH = "#{CONFIG_PATH}/Local"
-
 platform :ios do
   before_all do |lane, options|
-    unless lane == :test || options[:skip_bootstrap]
-      sh("pushd .. && ./bootstrap.sh --ci && popd")
-    end
-    # Verify clean repo by default
-    unless options[:ignore_git_status]
-      UI.message "Running git status check"
-      ensure_git_status_clean(show_uncommitted_changes: true, show_diff: true)
+    unless lane == :upload
+      sh("pushd .. && ./scripts/bootstrap.sh --ci && popd")
     end
   end
 
-  after_all do |lane, options|
-    # If no uncommitted changes existed before running `lane`, reset any potential fastlane changes
-    reset_git_repo(skip_clean: true) unless options[:ignore_git_status]
-
-    clean_icons()
-  end
-
-  error do
-    clean_icons()
-  end
-
   desc "Run Unit Tests"
   lane :test do |options|
     run_tests(
       project: "App/Client.xcodeproj",
       scheme: "Debug",
-      devices: options[:test_all] ? ["iPhone 14 (16.4)", "iPhone 14 (17.0)"] : ["iPhone 14 (16.4)"],
+      devices: ["iPhone 15"],
       code_coverage: true,
+      output_types: "junit",
+      output_files: "junit.xml",
       ensure_devices_found: true,
+      derived_data_path: "../../../out/DerivedData",
```

Comment:
nit: this could be a constant. easier to replace in the future and i see it in few places